taskDev12/src/main/java/checkers/Main.java

```diff
@@ -0,0 +1,39 @@
+package checkers;
+
+/**
+ * Calculate number of steps for checker from departure point to destination point.
+ */
+public class Main {
+  private static final String ENTER_DEPARTURE_POINT_ASK = "Enter departure position: ";
```

Comment:
Better to call ENTER_DEPARTURE_POINT_QYESTION instead of  ENTER_DEPARTURE_POINT_ASK